<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskCopyDialogClass</class>
 <widget class="QDialog" name="TaskCopyDialogClass">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>168</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>328</width>
    <height>168</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>328</width>
    <height>168</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Copy Tasks</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>How many copies should be created?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>166</x>
     <y>135</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Ok">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>135</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEditStartTime">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>38</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>the new start time of (the first) observation</string>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd hh:mm:ss</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxNrCopies">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>8</y>
     <width>60</width>
     <height>26</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>the number of copies of the (group) of observations</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Start time (first task):</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTimeStep">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Time step between tasks:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxNewStatus">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>98</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>set the status of the copies to...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Set status of copies to:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTimeStep">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>68</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>When copying a single observation:
the time between each copy

When copying a group of observations:
the time between each group of observations</string>
   </property>
   <property name="inputMask">
    <string>0000:00:00; </string>
   </property>
   <property name="text">
    <string>0000:00:00</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
